ci/test/04_install.sh

```diff
@@ -112,10 +112,13 @@ if [[ ${USE_MEMORY_SANITIZER} == "true" ]]; then
 fi
 
 if [[ "${RUN_TIDY}" == "true" ]]; then
-  CI_EXEC "mkdir -p ${BASE_SCRATCH_DIR}/iwyu/build/"
-  CI_EXEC "git clone --depth=1 https://github.com/include-what-you-use/include-what-you-use -b clang_14 ${BASE_SCRATCH_DIR}/iwyu/include-what-you-use"
-  CI_EXEC "cd ${BASE_SCRATCH_DIR}/iwyu/build && cmake -G 'Unix Makefiles' -DCMAKE_PREFIX_PATH=/usr/lib/llvm-14 ../include-what-you-use"
-  CI_EXEC "cd ${BASE_SCRATCH_DIR}/iwyu/build && make install $MAKEJOBS"
+  export DIR_IWYU="${BASE_SCRATCH_DIR}/iwyu/"
```

Comment:
Not sure, but it looks like a `/` should either be dropped here, or dropped in the other lines that use it. 
```suggestion
  export DIR_IWYU="${BASE_SCRATCH_DIR}/iwyu"
```

Reply:
thx changed. I don't think it makes a difference, though.